storage/test/files.test.js

```diff
@@ -15,7 +15,9 @@
 
 var proxyquire = require('proxyquire').noCallThru();
 var bucketName = 'foo';
-var fileName = 'test.txt';
+var fileName = 'test1.txt';
```

Comment:
Nit: should this be `srcFileName`? (To clearly differentiate it vs. the other `*FileName` variables.)